notebooks/official/tabular_workflows/tabnet_on_vertex_pipelines.ipynb

```diff
@@ -0,0 +1,1315 @@
+{
```

Comment:
<p>My understanding is that a notebook is supposed to be a streamlined experience. Here is your dataset, here are the values you need to set, here's the result. It's not an API and it doesn't need to describe all the possible transforms and their configurations. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/726/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Reply:
<p>I agree. However, since FTE is also in private preview, I think we should keep examples on how it can be used in the notebook until we can point to documentation.</p>

Reply:
<p>There will be a link to this at [1] once the latest GCPC release is out. The GCPC release is in progress. I will update here when the release is done and there is a link to use.</p><p> </p><p>[1] https://google-cloud-pipeline-components.readthedocs.io/en/google-cloud-pipeline-components-1.0.13/google_cloud_pipeline_components.experimental.automl.tabular.html</p><p> </p>

Reply:
This is the link to the transformations list: https://google-cloud-pipeline-components.readthedocs.io/en/google-cloud-pipeline-components-1.0.15/google_cloud_pipeline_components.experimental.automl.tabular.html#google_cloud_pipeline_components.experimental.automl.tabular.FeatureTransformEngineOp

Reply:
Thanks, updated the notebook.